### ISSUE TITLE
Replay not proper for some archived Facebook pages

### ISSUE THREAD

**Issue by peterchanws** [sul-dlss/was-pywb#82]
Thumbnail not generated properly for the following
https://argo-qa.stanford.edu/view/druid:xp669tj1535

Here is the druid for the wacz file:
https://argo-qa.stanford.edu/view/druid:tb025vw0473

**Comment by justinlittman** [comment]
Here is how the page renders in pywb:
![image.png](https://images.zenhubusercontent.com/5d8270f64def500001d12f23/34e822d5-619d-4480-b86a-4768d4e2d35f)

Given that it isn't really rendering properly, what is the expected screenshot?

**Comment by ndushay** [comment]
@lwrubel @peterchanws - I wonder if this ticket should become "get facebook archives to appear properly in pywb"

**Comment by lwrubel** [comment]
@peterchanws will add other examples

**Comment by peterchanws** [comment]
The reported case is one of the 2 wacz file registered. Here is the wacz file druid: 
https://argo-qa.stanford.edu/view/druid:tb025vw0473

This is the seed without problem: https://argo-qa.stanford.edu/view/druid:pk426cf3177

**Comment by lwrubel** [comment]
Side note: I notice when I click on the page in pywb, it then renders correctly: 
https://swap-qa.stanford.edu/was/20211013220228/https://www.facebook.com/artxcollective/

The wacz in the initial example also renders grayed out if replayed using replayweb.page. So it does not seem to be a pywb-specific issue. 

Having it grayed out when loaded means that the thumbnail generated once a seed is registered for https://www.facebook.com/artxcollective/ will also be grayed out. 

I created a wacz of the artxcollective page when I was not logged into Facebook. I then replayed it in replayweb.page and it is not grayed out. 

@peterchanws is the solution here to not capture Facebook pages while logged in? (This is better in general, when possible, for social media sites.)
